requirements.txt

```diff
@@ -2,10 +2,10 @@ anyio==3.6.2 ; python_version >= "3.8" and python_version < "4.0"
 attrs==21.4.0 ; python_version >= "3.8" and python_version < "4.0"
 backoff==1.11.1 ; python_version >= "3.8" and python_version < "4.0"
 bson==0.5.10 ; python_version >= "3.8" and python_version < "4.0"
-certifi==2022.9.24 ; python_version >= "3.8" and python_version < "4.0"
+certifi==2022.12.7 ; python_version >= "3.8" and python_version < "4.0"
 colorama==0.4.6 ; python_version >= "3.8" and python_version < "4.0" and sys_platform == "win32"
 h11==0.14.0 ; python_version >= "3.8" and python_version < "4.0"
-httpcore==0.16.1 ; python_version >= "3.8" and python_version < "4.0"
+httpcore==0.16.2 ; python_version >= "3.8" and python_version < "4.0"
 httpx==0.23.1 ; python_version >= "3.8" and python_version < "4.0"
```

Comment:
`poetry.lock` and `requirements.txt` is changed after I ran `make install`, I think that should be okay? @ddl-gabrielhaim